### PR TITLE
Fixed write_dot error for networkx>=1.11 in nipype/pipeline/utils.py

### PR DESCRIPTION
**Pull request by polosecki**
This fixes an incompatibility with networkx>=1.11 reported here:
https://github.com/nipy/nipype/issues/1350#issuecomment-181495121

### CODE REVIEW

File: nipype/pipeline/engine/utils.py [hunk 1]
```diff
@@ -1021,7 +1021,10 @@ def export_graph(graph_in, base_dir=None, show=False, use_execgraph=False,
                                suffix='.dot',
                                use_ext=False,
                                newpath=base_dir)
-    nx.write_dot(pklgraph, outfname)
+    if float(nx.__version__)<1.11:
```
Comment (mwaskom) [inline nipype/pipeline/engine/utils.py]: This won't work because, e.g., `1.9` is numerically greater than `1.11`.

Comment (polosecki) [inline nipype/pipeline/engine/utils.py]: True, how dumb

Comment (satra) [inline nipype/pipeline/engine/utils.py]: you can use: `LooseVersion(nx.__version__) < LooseVersion('1.11')`

Comment (mwaskom) [inline nipype/pipeline/engine/utils.py]: Is it necessary to do the version check? NX removed the top-level import, but did they move the function?

Comment (polosecki) [inline nipype/pipeline/engine/utils.py]: I changed it to do the proper version check suggested by @satra

File: nipype/pipeline/engine/utils.py [hunk 2]
```diff
@@ -1021,7 +1022,10 @@ def export_graph(graph_in, base_dir=None, show=False, use_execgraph=False,
                                suffix='.dot',
                                use_ext=False,
                                newpath=base_dir)
-    nx.write_dot(pklgraph, outfname)
+    if LooseVersion(nx.__version__) < LooseVersion('1.11'):                           
+        nx.write_dot(pklgraph, outfname)
+    else:
+        nx.drawing.nx_pydot.write_dot(pklgraph, outfname)
```
Comment (satra) [inline nipype/pipeline/engine/utils.py]: @mwaskom - it's possible that they changed the api here completely.

@polosecki - could you please check if a former version of nx has this function, in which case you don't need the `if ... else ...``` clause.

Comment (mwaskom) [inline nipype/pipeline/engine/utils.py]: It looks like the change just removed the direct import inside nx: https://github.com/networkx/networkx/pull/1929

**Comment by polosecki** [comment]
Now the change is done without checking nx version, since the difference with older nx version is only that they removed the direct import but the write_dot function was always defined in the same place.

**Comment by chrisfilo** [comment]
it seems backwards compatible with 1.10

**Pull request merged**